website/content/docs/api-clients/client-agent.mdx

```diff
@@ -771,6 +771,28 @@ Running a MacOS VM in Parallels Desktop with shared networking causes the system
 As a workaround, you can configure Parallels Desktop to use a different network mode.
 Select an alternative network configuration such as **Host-Only**, **Default Adapter**, or **Wi-Fi** instead of **Shared Network**.
 Refer to the Parallels Desktop documentation for more information.
+### Cisco AnyConnect VPN
+
+When you run the Cisco AnyConnect VPN at the same time as the Boundary Client Agent, DNS resolution may fail for Boundary aliases.
+
+**Root cause**:
+The Boundary Client Agent runs on the default IPv4 range `100.x.x.x`. Cisco AnyConnect VPN treats this range as a *secured range*, which forces DNS queries to resolve through the VPN instead of the Boundary Client Agent. As a result, Boundary aliases cannot be resolved.
```

Comment:
```suggestion
The Boundary Client Agent runs on the default IPv4 range `100.x.x.x`. Cisco AnyConnect VPN treats this range as a **secured range**, which forces DNS queries to resolve through the VPN instead of the Boundary Client Agent. As a result, Boundary aliases cannot be resolved.
```

Reply:
We use bold for emphasis instead of italics because italics can be difficult for some people to see.